lib/datasets/volume.rb

```diff
@@ -32,5 +32,9 @@ def to_s
       "#{namespace}.#{id} #{right} #{access_profile}"
     end
 
+    def hash
```

Comment:
ah, no, only explained in the commit for it (https://github.com/hathitrust/datasets/pull/51/commits/dafdd935ad521b78714f84060effc28200089216). I could add a comment in `Volume`